lib/lib-auto-generated/src/main/resources/schema/org/sagebionetworks/repo/model/schema/CreateOrganizationRequest.json

```diff
@@ -3,7 +3,7 @@
 	"properties": {
 		"organizationName": {
 			"type": "string",
-			"description": "Organization names must start and end with with a letter ([a-z]) and can contain digits [0-9] and periods \".\".  The name must be at least 3 characters and no more than 250 characters. Names are case insensitive. Names cannot contain the reserved word'sagebionetworks'"
+			"description": "An organization name must be one or more alphanumeric strings separated each separated by a dot [ <alaph_num>('.'<alpha_num>)*].  An alphanumeric string must start with letter followed by one or more letters or digits [a-z]([a-z0-9])*.  The name must be at least 6 characters and no more than 250 characters. Names are case insensitive. Names cannot contain the reserved word'sagebionetworks'"
```

Comment:
An organization name must be one or more alphanumeric strings each separated by a dot [ <alpha_num>('.'<alpha_num>)*].  An alphanumeric string must start with a letter followed by one or more letters or digits [a-z]([a-z0-9])*.  The name must be at least 6 characters and no more than 250 characters. Names are case insensitive. Names cannot contain the reserved word 'sagebionetworks'